echidna/core/smear.py

```diff
@@ -233,6 +237,81 @@ def random_gaussian_energy_spectra(self, true_spectrum):
         smeared_spectrum._raw_events = raw_events
         return smeared_spectrum
 
+
+    def random_gaussian_energy_spectra_parallel(self, true_spectrum,nProcs):
+        """ Smears the energy of a spectra object using multiprocessing  by generating
+          a number of random points from a Gaussian pdf generated
+          for that bin. The number of points generated is equivalent
+          to the number of entries in that bin.
+
+        Args:
+          true_spectrum (spectra): spectrum to be smeared
+
+        Returns:
+          A smeared spectra object.
```

Comment:
@Mark--S generally docs look ok. A small point on formatting, we are trying to move towards `Returns:` blocks of the form:
```
Returns:
  <return_type>: <description>
```
as this is formatted really nicely in the built docs. See the `DBIsotope` class for examples where this has been implemented.